frontend/src/test/scala/bloop/bsp/BspMetalsClientSpec.scala

```diff
@@ -300,7 +300,13 @@ class BspMetalsClientSpec(
 
   test("compile with semanticDB") {
     TestUtil.withinWorkspace { workspace =>
-      val `A` = TestProject(workspace, "A", dummyFooScalaAndBarJavaSources)
+
+      object JavacOptions {
+        // This will cause to use the forked javac compiler
```

Comment:
why it'll cause forking? Does every option cause forking?

Reply:
Yes, every `-J` option causes forking.

Reply:
If you add ` every -J option causes forking.` as a comment it'll be awesome

Reply:
Added!